### DISCUSSION TITLE
0.9 Update

### DISCUSSION THREAD

**Discussion by japanese-goblinn** [intellibar/main#20] [Ideas]
Hey there!

I just wanted to say that I'm really loving the new update you guys released. Great job! However, I do have a few points that I wanted to bring up:

1. It would be awesome if this window could also appear in dedicated windows. That way, it would be even more convenient for users like me.
<img width="324" alt="image" src="https://github.com/intellibar/main/assets/33631005/e4809219-5484-4050-9f6a-82ca70a8b6de">

2. Another suggestion I have is to allow for the window mentioned above and the "ask about selection" window to be rearranged to different positions and changed in size. This would give users more flexibility and customization options.

3. I've noticed a slight lag when opening the settings. It would be fantastic if you could look into optimizing that for a smoother experience.

4. Lastly, I came across a small issue with the rendering of the "open chat" menu. It would be great if you could take a look and fix it.
 <img width="322" alt="image" src="https://github.com/intellibar/main/assets/33631005/d79b6a26-b0f3-40b9-b45a-dd207683128b">

**Comment by japanese-goblinn** [comment]
UPD

5. Could you consider displaying usage estimates in the settings? It would be helpful for users to have that information available.
6. In the previous version, I loved how I could easily insert and edit previous prompts by pressing the arrow up key. Unfortunately, it doesn't seem to work anymore. Is there a possibility of bringing back this feature?
7. I can't seem to find the option to disable prompt completions in the settings anymore. Has it been removed?

**Comment by erusev** [comment]
Hi there, thanks for the suggestions and I'm glad that you like the update.

> I came across a small issue with the rendering of the "open chat" menu

We're on it.

> In the previous version, I loved how I could easily insert and edit previous prompts by pressing the arrow up key.

We'll try to bring it back.

> I can't seem to find the option to disable prompt completions in the settings anymore. Has it been removed?

Yes, we removed it because we felt it might not provide enough value, but we might be wrong.

> It would be awesome if this window could also appear in dedicated windows. That way, it would be even more convenient for users like me.

Can you tell me more about this? I'd like to know what your use case is for this.

**Reply by japanese-goblinn** [reply]
I've written about this in [this discussion](https://github.com/intellibar/main/discussions/16). Essentially, I prefer using fullscreen mode for all of my apps. However, switching to the main `IntelliBar` window forces me to change my current context when all I want is to quickly access ChatGPT.

**Reply by erusev** [reply]
Does the "ask abt selection" window work for you in fullscreen?

**Reply by japanese-goblinn** [reply]
Yeah, it does. But it's more about quickly ask for more question about last prompt or a random thing that is not necessary involves any selection.

**Reply by erusev** [reply]
I understand. Thanks for sharing your thoughts on this, I'll see what we can do.
